common/src/main/java/org/terracotta/angela/common/AngelaProperties.java

```diff
@@ -46,6 +46,10 @@
   // display Ignite logging (used to help debugging the behaviour of Angela)
   IGNITE_LOGGING("angela.igniteLogging", "false"),
 
+  // Define additional hostnames or ip addresses to be considered as local, separated by comma.
+  // For some testing, it is possible ot fake some local hostnames.
+  LOCAL_HOSTNAMES("angela.additionalLocalHostnames", ""),
```

Comment:
will allow to control the list of local hostnames.
Some tests are simulating faked local hostnames so we need to add them.
The alternatives is for angela to be ale to undertand a host file (or passed one to the jvm as an override). 
This is not a pragmatic solution.

Reply:
Why to expose this as property ?

Reply:
BEcause in tc-platform, for some system tests, we are overriding the hosts file for the JVM to include some fake hostnames:

`<jdk.net.hosts.file>${project.build.directory}/fake-hosts.txt</jdk.net.hosts.file>`

Angela needs to be able to know which hostname is "local". Currently, it is based on localhost, 127.0.0.1, ::1 and `InetAddress.getLocalHost().getHostName()`. The problem is that the latter only returns 1 value.

So this property is used to hint angela which additional hostnames have to be considered local, so that angela does not trigger the SSH remote agent installation...

Reply:
In a host file we can have for example entries such as:
`127.0.0.1 machine.local fajehostname foo bar baz`